src/components/ConnectWallet/TorSetup.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="px-3 pb-2">
+    <p class="font-weight-bold">How to install Tor on your system</p>
+    <b-form-select v-model="selectedOS" class="mb-3">
```

Comment:
Do we need this as a modal? Maybe it would make more sense just having it directly in the list of actions.

//cc @mayankchhabra what do you think?

Reply:
Tor setup is a system one-time action, so I don't think we should having it directly in the list of actions for each wallet. Currently, if Tor is not installed, the user just has to click to display the Tor setup modal, and if it is, there are no "useless" / redundant instructions in the list of actions. At least it is what I think, let's see @mayankchhabra's thoughts.

Reply:
Yeah I agree it would have made sense to include in the list of steps if setting up Tor on the system was always required, like setting up Orbot is always required for every new Android wallet for instance. But since that isn't the case, it makes more sense to exclude it from the steps.